app/components/TaskList/index.tsx

```diff
@@ -92,7 +92,7 @@ const TaskList: React.FC<Props> = ({
 
   const [stageDirtyFormChanges] = useStageDirtyFormChanges();
   useEffect(() => {
-    if (mode !== "view")
+    if (mode !== "view" && !router.query?.isRoutedFromNew)
```

Comment:
This is the best way I could think of to control the tasklist status. Previously, it was controlled by checking if the form was pristine (`newFormData` was null). Now that we're creating the project with rfp info (`newFormData: {fundingStreamRfpId: #}`, there's no way to tell if `newFormData` contains only rfp data because the project has just been created (in progress status), or because the user forgot to fill out other fields (attention required status).

Reply:
This seems to be a valid use of next router based on their docs. I wonder if there's an argument for there to be a new field to hold the `newFormStatus` to distinguish between `in progress status` and `just created`. This also could be a discussion in `1299 do we need form change is pristine`